CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ Note that Sockeye has checks in place to not translate with an old model that wa
 
 Each version section may have have subsections for: _Added_, _Changed_, _Removed_, _Deprecated_, and _Fixed_.
 
+## [1.18.59]
+- A flag is returned alongside the best metric indicating if training converged.
```

Comment:
I'd put this under an `### Added section`.

Reply:
Oh, it was eaten up by a merge, returned.